views/apartments/index.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>ApartmentManager.com</title>
+  <style>
+    h1, h2, h4 {
+      text-align: center;
+    }
+    a {
+      text-decoration: none;
+    }
+    p {
+      text-align: center;
+    }
+    input {
+      display: block;
+      margin-bottom: 10px;
+    }
+  </style>
```

Comment:
we didn't learn abotu it yet at the time of the sinatra class. But layouts could really help clean up this code base.